packages/orbit-components/src/primitives/ButtonPrimitive/README.md

```diff
@@ -54,21 +54,12 @@ Table below contains all types of the props available in ButtonPrimitive compone
 | padding          | `string`                   |            | The inner padding of the ButtonPrimitive.                                                                                                                      |
 | rel              | `string`                   |            | Specifies the rel of an element. [See Functional specs](#functional-specs)                                                                                     |
 | role             | `string`                   |            | Specifies the role of an element.                                                                                                                              |
-| size             | [`enum`](#enum)            | `"normal"` | The size of the ButtonPrimitive.                                                                                                                               |
 | spaceAfter       | `enum`                     |            | Additional `margin-bottom` after component. [See this docs](https://github.com/kiwicom/orbit/tree/master/packages/orbit-components/src/common/getSpacingToken) |
 | submit           | `boolean`                  | `false`    | If `true`, the ButtonPrimitive will have `type="submit"` attribute, otherwise `type="button"`.                                                                 |
 | title            | `string`                   |            | Adds `aria-label`.                                                                                                                                             |
 | tabIndex         | `string \| number`         | `"0"`      | Specifies the tab order of an element.                                                                                                                         |
 | width            | `string`                   |            | The width of the ButtonPrimitive. Can be any string - `100px`, `20%`.                                                                                          |
 
-### enum
-
-| size       |
-| :--------- |
-| `"small"`  |
-| `"normal"` |
-| `"large"`  |
-
```

Comment:
`size` needs to be removed from other READMEs of Button, ButtonLink and SocialButton as well.

Reply:
I can't actually delete that @silvenon. The `size` is still available on `Button`, `ButtonLink` and `SocialButton`. It was deleted only for the ButtonPrimitive, because it doesn't have any effect on the visual style. The related attributes to the `size` are calculated on the e.g. `Button` level.